test/rules/from-map-ts.js

```diff
@@ -0,0 +1,75 @@
+import AvaRuleTester from "eslint-ava-rule-tester";
+import test from "ava";
+import rule from "../../rules/from-map";
+
+const ruleTester = new AvaRuleTester(test, {
+    parser: require.resolve('@typescript-eslint/parser'),
+    parserOptions: {
+        ecmaVersion: 2020,
+        sourceType: 'module'
+    }
+});
+
+ruleTester.run('from-map', rule, {
```

Comment:
I wonder if the actual rule tests should be shared with a helper module, so adding a test for the rule automatically checks for both parsers.